packages/chrome/stories/examples/Sheet.tsx

```diff
@@ -0,0 +1,156 @@
+              artichoke. Nori grape silver beet broccoli kombu beet greens fava bean potato quandong
+              celery. Corn amaranth salsify bunya nuts nori azuki bean chickweed potato bell pepper
+              artichoke. Nori grape silver beet broccoli kombu beet greens fava bean potato quandong
+              celery. Corn amaranth salsify bunya nuts nori azuki bean chickweed potato bell pepper
+              artichoke. Nori grape silver beet broccoli kombu beet greens fava bean potato quandong
+              celery. Corn amaranth salsify bunya nuts nori azuki bean chickweed potato bell pepper
+              artichoke. Nori grape silver beet broccoli kombu beet greens fava bean potato quandong
+              celery.
+            </SM>
+          </Paragraph>
+          <Paragraph>
+            <SM tag="span">
+              Candy cane winter Malaysian sleepy morning tea refreshing cucumber splash chilies
+              dessert blueberry pops avocado fresh bananas lingonberry. Tasty grenadillo peach
+              strawberry mango cayenne hummus Caribbean red habanero edamame hummus bento box
+              pumpkin butternut mix chili asian pear green tea lime entree maple orange tempeh miso
+              dressing alfalfa sprouts winter cherry black bean wraps toasted hazelnuts lavender
+              lemonade. Pesto banh mi salad rolls peanut butter one bowl apples mocha chocolate cool
+              off lemon Bolivian rainbow pepper avocado dressing drizzle black beans cinnamon mangos
+              garlic sriracha noodles Thai sun pepper creamiest lentils oranges soba noodles. Lime
+              mango crisp couscous paprika pine nuts kung pao pepper vitamin glow picnic hearty
+              sweet potato peaches appetizer green grapes tabasco pepper red amazon pepper Mexican
+              fiesta dark chocolate coriander Sicilian pistachio pesto ginger tofu basil chocolate.
+            </SM>
+          </Paragraph>
+        </Sheet.Body>
+        <Sheet.Footer isCompact={isCompact}>
+          {isCompact ? (
+            <Sheet.FooterItem>
+              <Anchor href="/#">Footer</Anchor>
+            </Sheet.FooterItem>
+          ) : (
+            <>
+              <Sheet.FooterItem>
+                <Button isBasic>Action</Button>
+              </Sheet.FooterItem>
+              <Sheet.FooterItem>
+                <Button isPrimary onClick={() => updateArgs({ isOpen: false })}>
+                  Close
+                </Button>
+              </Sheet.FooterItem>
+            </>
+          )}
+        </Sheet.Footer>
+        <Sheet.Close onClick={() => updateArgs({ isOpen: false })} />
+      </Sheet>
+    </ThemeProvider>
+  );
+};
+
+SheetStory.storyName = 'Standalone Sheet';
+
+SheetStory.parameters = {
+  layout: 'centered'
+};
+
+SheetStory.args = {
+  rtl: false,
+  isOpen: true,
+  isAnimated: true,
+  focusOnMount: false,
+  restoreFocus: false,
+  isCompact: false,
+  placement: 'end'
```

Comment:
I don't notice any difference in the UI/UX when the `placement` knob is toggled. How is the UI/UX different when the `placement` toggle is switched to `start` (vs `end`)?

Reply:
The `placement` prop is to allow the consumer to explicitly state where the `Sheet` will be positioned, therefore the borders will react accordingly to prevent double bordering. 

When the `Sheet` is positioned on the `right`, therefore placement will be `end`
![image](https://user-images.githubusercontent.com/30312964/140565345-f825d3d9-4a45-4fed-a96c-5eb41bbc2b92.png)

When the `Sheet` is positioned on the `left`, therefore placement will be `start`
![image](https://user-images.githubusercontent.com/30312964/140565381-8643cba7-a4fd-4f63-aa30-2849e2ab6106.png)

Reply:
It shifts over the border to expect the `Sheet` (which is very subtle) to be placed on that respective placement.